packages/cli/commands/import/workflow.ts

```diff
@@ -111,6 +120,11 @@ export class ImportWorkflowsCommand extends Command {
 							});
 						}
 
+						if (Object.prototype.hasOwnProperty.call(workflow, 'tags')) {
+							const tags = await Db.collections.Tag.find();
+							await setTagsForImport(transactionManager, workflow, tags);
+						}
```

Comment:
We are querying the DB even if `tags` is not an array or is an empty array. We should use the `hasTags` helper check here and remove it from `setTagsForImport`. Same elsewhere.